src/sentry/tasks/update_user_reports.py

```diff
@@ -28,16 +30,23 @@ def update_user_reports(**kwargs):
     updated_reports = 0
     samples = None
 
+    MAX_EVENTS = kwargs.get("max_events", 5000)
```

Comment:
Is this going to end up too high?

Reply:
Some napkin math shows that if the entire query was just event IDs, we could support around 13,000. I figure at this limit (5000) we could in theory have the entire list of event IDs twice and still have plenty of space for the rest of the query.